week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,18 @@
 */
 
 function calculateGrade(mark) {
-
+  if (80 <= mark && mark <= 100) {
+    return "A";
+  }
+  if (60 <= mark && mark <= 80) {
```

Comment:
You can use `else if` when you only want to check a second conditional if the first one fails. Like this:

```
if (80 <= mark) {
  return "A";
} else if (60 <= mark) {
  return "B";
}
```

Notice how doing it this way, I didn't have to set the upper limit for a "B". If it's greater than `80`, it will return "A" and stop. Otherwise, it will evaluate the second conditional. At that point, I already know that it's less than `80`, so I don't have to check it.

For example, what do you think would happen if someone got a `110` mark? What grade would be returned?